api/napi/index.ts

```diff
@@ -196,15 +218,27 @@ interface Callback {
     setHandler(cb: any): void;
 }
 
+/**
+ * Represents an errors that can be emitted by the compiler.
+ */
 export class CompilerError extends Error {
```

Comment:
I realise that this type has the wrong name. It's not `CompilerError`, it should be `CompileError`. (It's not an error of the compiler, it's an error that occurred during compilation of the .slint file :-). Please rename.

Reply:
done